mlir/utils/jenkins/Jenkinsfile

```diff
@@ -723,6 +730,10 @@ pipeline {
                                 selector: lastSuccessful(),\
                                 target: 'MIOpen/build/MIOpenUserDB'
                                 sh 'ls MIOpen/build/MIOpenUserDB'
+                                // Move GEMM tuning DB out of the way if it exists
+                                catchError {
+                                    sh "mv MIOpen/build/MIOpenUserDB/*.tsv ${WORKSPACE}"
```

Comment:
Why do you need to move the db out of the directory?

Reply:
Because we'll be `rm`-ing that directory to get untuned MIOpen numbers

Reply:
From the doc, `catchError` will mark the build as `failure`, which will trigger node reboot in our nightly CI's. Is this the expected behavior of Gemm tuning?

Reply:
I though the point of `catchError` was to _not_ mark the build as a failure, so that we don't reboot if the weekly CI doesn't have tuning info